oaknational/Oak-Web-Application
# fix: 1085 fix border box gap to match designs

## Description

- Amend the box border gap to match the designs
- This `gapPosition` property is used with multiple boxes across the site and this is the easiest solution I found to make sure it doesn't get broken in other places
- If we want a cleaner solution for those box borders I will need to sit with Stuart to discuss how and where we want to use them across the site and refactor it because there are few different border gaps at the moment (not only % of the gap differs but also the position of the gap).
- the previous implementation only allows to set one border with the gap and for this case we need two (both bottom and right borders to have a gap)

## Issue(s)

Fixes #1085

## How to test

1. Go to `/beta/teachers/key-stages/ks1/subjects/art/units/sculpture-93b1` and `/beta/teachers/key-stages/ks1/subjects/art/units`
2. Check the borders gap
3. Check if those border boxes on `/beta/teachers/` still look good too

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
